homeassistant/components/light/yeelight.py

```diff
@@ -67,14 +67,13 @@ def _wrap(self, *args, **kwargs):
 
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
-    """Setup the Yeelight bulbs."""
+    """Set up the Yeelight bulbs."""
     lights = []
     if discovery_info is not None:
         _LOGGER.debug("Adding autodetected %s", discovery_info['hostname'])
 
-        # not using hostname, as it seems to vary.
-        name = "yeelight_%s_%s" % (discovery_info["device_type"],
-                                   discovery_info["properties"]["mac"])
+        name = 'yeelight_{}_{}'.format(
+            discovery_info['device_type'], discovery_info['properties']['mac'])
```

Comment:
Imo format string changes should be discussed separately. In the future (with 3.6) it'd be nice to use f-strings directly, however for now I don't see what good does this change do (when not naming the placeholders, that is)?